content/docs/user-guide/networking/settings.md

```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
 ---
 
 ## Overview
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+## Overview
+This page documents [console variables](/docs/user-guide/appendix/cvars) and other settings that control Networking and Multiplayer behavior.
+
+## Networking commands
+These are special console commands that control the flow of networked and multiplayer games. 
+
+| Setting                | Description                                                                                | Default | Notes |
+|------------------------|--------------------------------------------------------------------------------------------|---------|-------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to.                  |||                                                                      ||
+| connect                | Opens a multiplayer connection to a host                                                   |||
+| disconnect             | Disconnects any open multiplayer connections                                               ||| 
+| LoadLevel              | Unloads the current level and loads a new one with the given asset name.                   ||| 
```

Comment:
Ah, nice call adding loadlevel. It's not multiplayer specific, but basically essential to anyone working with multiplayer (or looking at this page)